client/my-sites/email/email-management/home/email-plan-warnings.jsx

```diff
@@ -36,26 +30,17 @@ class EmailPlanWarnings extends React.Component {
 	renderCTAForTitanUnusedMailboxes() {
 		const { currentRoute, domain, selectedSiteSlug, translate } = this.props;
 
-		const showExternalControlPanelLink = ! isEnabled( 'titan/iframe-control-panel' );
-		const controlPanelUrl = showExternalControlPanelLink
-			? emailManagementTitanControlPanelRedirect( selectedSiteSlug, domain.name, currentRoute, {
-					context: TITAN_CONTROL_PANEL_CONTEXT_CREATE_EMAIL,
-			  } )
-			: emailManagementManageTitanAccount( selectedSiteSlug, domain.name, currentRoute, {
-					context: TITAN_CONTROL_PANEL_CONTEXT_CREATE_EMAIL,
-			  } );
+		const setupMailboxUrl = emailManagementTitanSetupMailbox(
+			selectedSiteSlug,
+			domain.name,
+			currentRoute
+		);
 
 		return (
-			<Button
-				compact
-				primary
-				href={ controlPanelUrl }
-				target={ showExternalControlPanelLink ? '_blank' : null }
-			>
-				{ translate( 'Activate Mailbox', 'Activate Mailboxes', {
-					count: getMaxTitanMailboxCount( domain ) - getConfiguredTitanMailboxCount( domain ),
-				} ) }
-				{ showExternalControlPanelLink && <Gridicon icon="external" /> }
+			<Button compact primary href={ setupMailboxUrl }>
+				{ 'en' === getLocaleSlug() || i18nCalypso.hasTranslation( 'Set up mailbox' )
+					? translate( 'Set up mailbox' )
+					: translate( 'Activate Mailbox' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Activate Mailbox', 'Activate Mailboxes' )` *ES Score: 17*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/use-internal-mailbox-creation-flow)